eth2deposit/settings.py

```diff
@@ -1,6 +1,9 @@
 from typing import Dict, NamedTuple
 
 
+DEPOSIT_CLI_VERSION = "0.4.0"
```

Comment:
I built a very complicated solution in #127 with a perl/regex version string replacer, but there is a much simpler solution I didn't think to try. 🙃 

```suggestion
import pkg_resources


DEPOSIT_CLI_VERSION = pkg_resources.require("eth2deposit")[0].version
```